app/uk/gov/hmrc/agentsubscriptionfrontend/models/Address.scala

```diff
@@ -137,7 +136,4 @@ object Address{
 
     OFormat[Address](reads, formatAddressValue)
   }
-
-  val renderErrors: PostCodeError => String = (errors: PostCodeError) => errors.foldLeft("")(_+", "+_).substring(1)
```

Comment:
I think my comments still apply - where is the unit test for `renderErrors` and does it need to be a `val`?